### ISSUE TITLE
After Update to 9.0 shared items are no longer displayed

### ISSUE THREAD

**Issue by loripino21** [owncloud/core#23909]
I had installed OC 8.2.1. I upgraded to 9.0.0. Now sections shared with me, shared with others, shared with link are empty. Even if I share a new folder or link, it doesn't appear.

### Steps to reproduce
1. Update OC 8.2.1 to OC 9.0.0

### Expected behaviour
Shared folders and link should show up

### Actual behaviour
All shared folders/shared links lists are empty

### Server configuration
**Operating system**: xUbuntu 15.04

**Web server:** Apache 2

**Database:** MySQL 5.6

**PHP version:** 5.6.4

**ownCloud version:** 9.0.0

**Updated from an older ownCloud or fresh install:** updated from OC 8.2.1

**Where did you install ownCloud from:** official package

**Signing status (ownCloud 9.0 and above):** 

```
Login as admin user into your ownCloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.

No errors have been found.
```

**List of activated apps:**

```
Only default and official ones, plus Tasks.
```

**The content of config/config.php:**

```
<?php
$CONFIG = array (
  'instanceid' => '',
  'passwordsalt' => '',
  'trusted_domains' => 
  array (
    0 => 'cloud..it',
    1 => '192.168.0.20',
    2 => '192.168.0.31',
  ),
  'overwritehost' => 'cloud..it',
  'datadirectory' => '/mnt/owncloud',
  'dbtype' => 'mysql',
  'version' => '9.0.0.19',
  'dbname' => 'owncloud',
  'dbhost' => '192.168.0.100',
  'dbtableprefix' => 'oc_',
  'dbuser' => 'oc_admin',
  'dbpassword' => '',
  'installed' => true,
  'loglevel' => 1,
  'theme' => '',
  'maintenance' => false,
  'secret' => '',
  'trashbin_retention_obligation' => 'auto',
  'memcache.local' => '\\OC\\Memcache\\Redis',
  'filelocking.enabled' => 'true',
  'memcache.distributed' => '\\OC\\Memcache\\Redis',
  'memcache.locking' => '\\OC\\Memcache\\Redis',
  'redis' => 
  array (
    'host' => 'localhost',
    'port' => 6379,
    'timeout' => 0,
    'dbindex' => 0,
  ),
  'appstore.experimental.enabled' => true,
);
```

**Are you using external storage, if yes which one:** local, mounted smb storage

**Are you using encryption:** no

**Are you using an external user-backend, if yes which one:** no

### Client configuration
**Browser:** Any

**Operating system:** Windows

### Logs
Nothing revelant.

#### Web server error log
No  revelant data

#### ownCloud log (data/owncloud.log)
no revelant data

**Comment by RealRancor** [comment]
@loripino21 Please update to oC 9.0.1 (released a few days ago) before reporting issues which might be already fixed in newer versions.

**Comment by loripino21** [comment]
I tried to update via the updater app. But running updater/application.php  I got "Current version 9.0.0.19. No updates found online". Should I open a new issue for it?

**Comment by RealRancor** [comment]
> Should I open a new issue for it?

Nope, thats not needed. 9.0.1 is not available via the updater yet and might be delayed a little bit. Once some one thinks it is stable enough it will be available via the updater app.

For now please do a manual update.

**Comment by enoch85** [comment]
I can confirm that this happened in 9.0 but it's fixed in 9.0.1, at least for me.

**Comment by DeepDiver1975** [comment]
> I can confirm that this happened in 9.0 but it's fixed in 9.0.1, at least for me.

case closed
